src/lib/cache/subCache.cpp

```diff
@@ -1260,7 +1260,7 @@ extern bool noCache;
 *
 * If 'errors' == 0, then the subscription is marked as non-erroneous.
 */
-void subCacheItemNotificationErrorStatus
+void subNotificationErrorStatus
```

Comment:
The `subCache` token as part of the name was misleading. The function also does its work in noCache mode (see the `if (noCache)` part inside the function body).

NTC (just informative)